tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/set_ice_contacts.py

```diff
@@ -0,0 +1,45 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from marionette_driver import expected, By, Wait
+
+from gaiatest import GaiaApps
+from gaiatest.apps.base import Base
+from gaiatest.apps.contacts.app import Contacts
+from gaiatest.form_controls.header import GaiaHeader
+
+
+class SetIceContacts(Base):
+
+    _toggle_locator = (By.CSS_SELECTOR, 'gaia-switch[name="ice-contact-1-enabled"]')
+    _select_ice_contact_1_locator = (By.ID, 'select-ice-contact-1')
+    _header_locator = (By.ID, 'ice-settings-header')
+    _ice_header_locator = (By.ID, 'ice-header')
+    _ice_contact = (By.CSS_SELECTOR, 'ol#ice-group')
+
+    def __init__(self, marionette):
+        Base.__init__(self, marionette)
+        GaiaApps(marionette).switch_to_displayed_app()
```

Comment:
Hrm, if you're doing that here, you don't need to do that in other places where you instantiate a SetIceContacts(self.marionette) object.